sport/app/cricket/feed/CricketThrottler.scala

```diff
@@ -1,13 +1,14 @@
 package cricket.feed
 
+import org.apache.pekko.NotUsed
+import org.apache.pekko.pattern.{ask, pipe}
+import org.apache.pekko.stream.{CompletionStrategy, Materializer, OverflowStrategy, ThrottleMode}
```

Comment:
```suggestion
import org.apache.pekko.stream.{CompletionStrategy, Materializer => PekkoMaterializer, OverflowStrategy, ThrottleMode}
```

Should we do it this way for consistency with [this file](https://github.com/guardian/frontend/pull/26563/files#diff-66b57af6c7eb5fd888ff96c4445a5ff50f269727e37c91e5258be32393e830fcR18) and `PekkoActorSystem`?